src/probnum/linalg/solvers/stopping_criteria/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Stopping criteria of probabilistic linear solvers determining termination."""
```

Comment:
Somehow this sentence souds weird to me. How about

```suggestion
"""Stopping criteria for probabilistic linear solvers"""
```